chapters/hpmor-chapter-001.tex

```diff
@@ -1,176 +1,175 @@
 
-His mother looked down at him and smiled. “Thank you, Harry. But—” her head rose back up to stare at her husband. “I don’t want to win an argument with your father. I want my husband to, to listen to his wife who loves him, and trust her just this once—”
+His mother looked down at him and smiled. “Thank you, Hari. But—” her head rose back up to stare at her husband. “I don’t want to win an argument with your father. I want my husband to, to listen to his wife who loves him, and trust her just this once—”
 
-Harry closed his eyes briefly. \emph{Hopeless.} Both of his parents were just hopeless.
+Hari closed his eyes briefly. \emph{Hopeless.} Both of his parents were just hopeless.
 
 Now his parents were getting into one of \emph{those} arguments again, one where his mother tried to make his father feel guilty, and his father tried to make his mother feel stupid.
 
-“I’m going to go to my room,” Harry announced. His voice trembled a little. “Please try not to fight too much about this, Mum, Dad, we’ll know soon enough how it comes out, right?”
+“I’m going to go to my room,” Hari announced. His voice trembled a little. “Please try not to fight too much about this, Mum, Dad, we’ll know soon enough how it comes out, right?”
 
-“Of course, Harry,” said his father, and his mother gave him a reassuring kiss, and then they went on fighting while Harry climbed the stairs to his bedroom.
+“Of course, Hari,” said his father, and his mother gave him a reassuring kiss, and then they went on fighting while Hari climbed the stairs to his bedroom.
 
 He shut the door behind him and tried to think.
 
 The funny thing was, he \emph{should} have agreed with Dad. No-one had ever seen any evidence of magic, and according to Mum, there was a whole magical world out there. How could anyone keep something like that a secret? More magic? That seemed like a rather suspicious sort of excuse.
 
 It should have been a clean case for Mum joking, lying or being insane, in ascending order of awfulness. If Mum had sent the letter herself, that would explain how it arrived at the letterbox without a stamp. A little insanity was far, far less improbable than the universe really working like that.
 
-Except that some part of Harry was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogwarts School of Witchcraft and Wizardry.
+Except that some part of Hari was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogwarts School of Witchcraft and Wizardry.
```

Comment:
```suggestion
Except that some part of Hari was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogŭartso School of Witchcraft and Wizardry.
```